app/fullService/api/buildTransaction.ts

```diff
@@ -58,12 +58,8 @@ const buildTransaction = async ({
 
   // TODO fix type, right now it just matches what the component is expecting
   const totalValueConfirmation = txProposal.outlayList
-    .map((outlay) => {
-      return BigInt(outlay.value);
-    })
-    .reduce((acc, cur) => {
-      return acc + cur;
-    });
+    .map((outlay) => BigInt(outlay.value))
+    .reduce((acc, cur) => acc + cur, BigInt(0));
```

Comment:
The names were already there: `acc` for accumulator and `cur` for current. I added the initial value because not including it will throw an error if the array is empty.